OsmAnd/src/net/osmand/plus/mapcontextmenu/editors/SelectCategoryDialogFragment.java

```diff
@@ -79,14 +78,18 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
 		if (gpxFile != null) {
 			if (gpxCategories != null) {
 				for (Map.Entry<String, Integer> e : gpxCategories.entrySet()) {
-					String categoryName = e.getKey();
-					addCategory(activity, ll, categoryName, e.getValue());
+					if (!e.getKey().equals(getContext().getString(R.string.personal_category_name))) {
```

Comment:
Why compare with localized name? Is SelectCategoryDialogFragment used for favs only?